Add requirement for review costs management (#920)
publiccodenet/standard: criteria/welcome-contributors.md

* Confirm that it is possible to submit suggestions for changes to the codebase.
* Confirm there are contribution guidelines.
* Confirm that the codebase governance is clearly explained, including how to influence codebase governance.

> * Check that contributing guidelines covers who is expected to cover the costs of reviewing contributions.

Reviewer: ```suggestion
* Check that contributing guidelines cover who is expected to cover the costs of reviewing contributions.
```

* Make sure that the documentation of the governance includes the current process for how to make changes to the governance.
* If the community has some consensus about how the governance should change, then include those ideas stated as ambitions in the documentation.

> * Describe how the codebase community wants to handle the review of incoming contributions. If needed, secure and allocate budget for that process.

Reviewer: ```suggestion
* If needed, make sure you have allocated budget for the contributions review process as agreed by the codebase community.
```
